### ISSUE TITLE
"The library has been modified by another program"

### ISSUE THREAD

**Issue by krantisaran** [JabRef/jabref#6618]
JabRef version  on 

- [ ] **Mandatory**: I have tested the latest development version from http://builds.jabref.org/master/ and the problem persists

Whenever I make any change to an entry in my database, I get a "The library has been modified by another program" message. When I click Review Changes and then Accept Changes, a second later the same message will reappear over and over. If I edit another entry, then I get the same message regarding the first entry I edited, and it deletes the second! This cycle just keeps going whenever I make any changes. I use Dropbox. The problem persists in the latest development version. 

I have used JabRef and Dropbox together for almost 15 years now and never had a problem, but this bug has made it next to impossible to use JabRef.

**Comment by Siedlerchr** [comment]
Duplicate of #5870 #6525 #6420

**Comment by wujastyk** [comment]
JabRef 5.4--2021-11-15--6293196
Linux 5.4.0-90-generic amd64 
Java 16.0.2 
JavaFX 17.0.1+1

Sorry to say, I just got this again today.  It's a curse.  

The first time I got the message, the display of differences showed literally no difference at all between old and new.  Not a single character.  The second time, it showed only a change in modificationdate:

![image](https://user-images.githubusercontent.com/762246/141838195-fb60ecb3-e2fb-4375-a03b-a4caefb67bfc.png)
